neqo-transport/src/connection/tests/mod.rs

```diff
@@ -208,17 +210,19 @@ fn handshake_with_modifier(
         if should_ping {
             a.test_frame_writer = Some(Box::new(PingWriter {}));
         }
+        println!(
+            "Processing {} at {:?}ms",
+            a.role(),
+            (now - start).as_millis()
+        );
         let output = a.process(input.as_ref(), now).dgram();
         if should_ping {
             a.test_frame_writer = None;
             did_ping[a.role()] = true;
         }
-        assert!(had_input || output.is_some());
-        if let Some(d) = output {
-            input = modifier(d);
-        } else {
-            input = output;
-        }
+        // TODO
+        // assert!(dbg!(had_input) || dbg!(output.is_some()));
```

Comment:
Can we put this back?

Reply:
This will need an update. In the case of an ECN blackhole a server would neither receive an input, nor would it produce any output.

Reply:
OK, we should do that.